detection_rules/rule.py

```diff
@@ -269,6 +269,29 @@ class EQLRuleData(QueryRuleData):
     type: Literal["eql"]
     language: Literal["eql"]
 
+    @cached_property
+    def max_span(self) -> Optional[eql.ast.TimeRange]:
+        """Maxspan value for sequence rules if defined."""
+        if eql.utils.get_query_type(self.ast) == 'sequence' and hasattr(self.ast.first, 'max_span'):
+            return self.ast.first.max_span
+
+    @cached_property
+    def look_back(self) -> Union[eql.ast.TimeRange, Literal['unknown']]:
+        """Lookback value of a rule."""
+        # https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#date-math
+        from_ = self.from_ or 'now-6m'
+        from_ = from_.split('-')
+        to = self.to
+
+        if len(from_) != 2 or from_[0] != 'now' or to not in (None, 'now'):
```

Comment:
The problem is that this would only currently support a `from` defined as `now-N` format and a `to` of `now` or undefined, whereas those fields allow full datemath

Reply:
true, but I think this is okay.